packages/dev/test/Flows.test.ts

```diff
@@ -284,6 +315,18 @@ options.forEach(params => {
       })
     }
 
+    async function fixTxDetails (txDetails: any, relayProvider: RelayProvider): Promise<void> {
+      if (params.relay) {
+        const { maxFeePerGas, maxPriorityFeePerGas } = await relayProvider.calculateGasFees()
```

Comment:
it makes more sense that calculateGasFees will return { mxaFeePerGas?, maxPrio?, gasPrice? }
then this "fix" will simply do:
`{ ... await relayProvider.calculateGasFees() }`

(it will help users, though, but will complicate the test, it will require an extra param "forceLegacy", which means to return gasPrice even in type-2 mode)
it is better for users in that it will not require this strange and confusing `"gasPrice = maxPriorityFeePerGas"`

Reply:
Resolved over the phone